src/code/z_scene.c

```diff
@@ -327,17 +327,17 @@ void func_80098C24(GlobalContext* globalCtx, SceneCmd* cmd) {
 
 // Scene Command 0x0E: Transition Actor List
 void func_80098C68(GlobalContext* globalCtx, SceneCmd* cmd) {
-    globalCtx->nbTransitionActors = cmd->transiActorList.num;
-    globalCtx->transitionActorList = SEGMENTED_TO_VIRTUAL(cmd->transiActorList.segment);
+    globalCtx->doorCtx.numActors = cmd->transiActorList.num;
+    globalCtx->doorCtx.actorList = SEGMENTED_TO_VIRTUAL(cmd->transiActorList.segment);
 }
 
-void func_80098CBC(GlobalContext* globalCtx, u8* nbTransitionActors) {
-    *nbTransitionActors = 0;
+void Door_Init(GameState* state, DoorContext* doorCtx) {
```

Comment:
In #857 you used `Actor_InitContext`, maybe `Door_InitContext` would be appropriate here too, it would help not mistaking this function for `Actor_SpawnTransitionActors`

Reply:
yeah good idea, i think that every "context" should be named consistently, `InitContext` is fine i think